### DISCUSSION TITLE
Wrap long lines in unified diff view

### DISCUSSION THREAD

**Discussion by olatoft** [community/community#8497] [Pull Requests] [answered]
In "Files changed", the split diff view will wrap lines which are longer than the screen is wide. Could this be added as an option for the unified diff view as well?

**Comment by Marcono1234** [comment, marked as answer]
Looks like long lines are in the diff view are automatically wrapped now, not sure how long that is already the case.

However, for regular (non-diff) code view lines are not wrapped, see also [this feedback comment](https://github.com/orgs/community/discussions/48301#discussioncomment-5111826).
